src/__test__/components/ContentWrapper.test.jsx

```diff
@@ -71,7 +71,12 @@ describe('ContentWrapper', () => {
     expect(sider.length).toEqual(1);
 
     const menus = wrapper.find(Menu).children().find(Item);
-    expect(menus.length).toEqual(4);
+
+    // Menu item renders twice to support HOC usage (?)
+    // https://ant.design/components/menu/#Why-Menu-children-node-will-render-twice
+    const visibleMenuLength = menus.length / 2;
```

Comment:
After upgrading to new antd, Menu.Item is rendered twice : 1 visible, and 1 invisible for ARIA support. Therefore, to get the real number of visible items, this value is divided by 2. [more info](https://ant.design/components/menu/#Why-Menu-children-node-will-render-twice)